src/SignalR/common/Shared/ClientResultsManager.cs

```diff
@@ -23,7 +23,7 @@ public Task<T> AddInvocation<T>(string connectionId, string invocationId, Cancel
             var tcs = (TaskCompletionSourceWithCancellation<T>)state;
             if (completionMessage.HasResult)
             {
-                tcs.SetResult((T)completionMessage.Result);
+                tcs.SetResult((T)completionMessage.Result!);
```

Comment:
CC @BrennanConroy this is to preemptively fix https://github.com/dotnet/installer/pull/15151#issuecomment-1371385658, which we'll see once we ingest the new SDK from that PR